grails-app/domain/org/pih/warehouse/core/Location.groovy

```diff
@@ -306,6 +306,16 @@ class Location implements Comparable<Location>, java.io.Serializable {
         return !onHold
     }
 
+    LocationStatus getStatus() {
+        if (organization) {
+            if (active && organization.active) {
+                return LocationStatus.ENABLED
+            }
+            return LocationStatus.DISABLED
+        }
+        return active ? LocationStatus.ENABLED : LocationStatus.DISABLED
```

Comment:
I think this might be more readable. 
```
   if (organization) {
      return active && organization.active ? LocationStatus.ENABLED : LocationStatus.DISABLED
   } else {        
       return active ? LocationStatus.ENABLED : LocationStatus.DISABLED
   }
```

Reply:
Will correct it to this:
```
if (organization) {
    return (active && organization.active) ? LocationStatus.ENABLED : LocationStatus.DISABLED
}
return active ? LocationStatus.ENABLED : LocationStatus.DISABLED
```
You don't need to use `else` here - if the `return` from if is executed, it doesn't go any further, so it doesn't execute `return` from the bottom